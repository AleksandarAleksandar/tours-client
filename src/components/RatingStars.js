import React from 'react'

export default function RatingStars(props) {
  let number = 0;
  if (props.rating > 0) {
    number = props.rating
  }
  let jsxStars = [];
  for (let i = 1; i <= number; i++) {
    jsxStars.push(
      <div key={i} className="star"></div>
    )
  }
  return (
    <div className="stars">
      {jsxStars}
    </div>
  )
}
