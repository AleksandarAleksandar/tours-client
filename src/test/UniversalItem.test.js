import React from 'react'
import { shallow } from 'enzyme'
import RatingStars from './../components/RatingStars'

// console.log(shallow(<RatingStars />));
// it('expect to render Rating stars', () => {
//   expect(shallow(<RatingStars />).length).toEqual(1)
// })

it('expect to render Rating stars', () => {
  expect(shallow(<RatingStars />).length).toMatchSnapshot(1)
})
