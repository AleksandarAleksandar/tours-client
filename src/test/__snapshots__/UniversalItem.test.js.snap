// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Rating stars 1`] = `1`;
